Index: 04. OOP/01.First Steps in OOP/02. Exercise/project/trainer.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04. OOP/01.First Steps in OOP/02. Exercise/project/trainer.py b/04. OOP/01.First Steps in OOP/02. Exercise/project/trainer.py
--- a/04. OOP/01.First Steps in OOP/02. Exercise/project/trainer.py	(date 1625900336043)
+++ b/04. OOP/01.First Steps in OOP/02. Exercise/project/trainer.py	(date 1625900336043)
@@ -1,6 +1,5 @@
 from typing import List
-
-from prject.pokemon import Pokemon
+from project.pokemon import Pokemon
 
 
 class Trainer:
@@ -11,8 +10,9 @@
     def add_pokemon(self, pokemon):
         if pokemon in self.pokemons:
             return "This pokemon is already caught"
-        self.pokemons.append(pokemon)
-        return f"Caught {pokemon.pokemon_details(pokemon)}"
+        else:
+            self.pokemons.append(pokemon)
+            return f"Caught {Pokemon.pokemon_details()}"
 
     def release_pokemon(self, pokemon_name):
         for pokemon in self.pokemons:
@@ -35,19 +35,20 @@
             print_solution += f'- {pokemon.pokemon_details()}{nl}'
         return print_solution
 
-pokemon = Pokemon("Pikachu", 90)
-print(pokemon.pokemon_details())
-trainer = Trainer("Ash")
-print(trainer.add_pokemon(pokemon))
-second_pokemon = Pokemon("Charizard", 110)
-second3_pokemon = Pokemon("Charizard3", 330)
-second4_pokemon = Pokemon("Charizard4", 440)
-second5_pokemon = Pokemon("Charizard5", 555)
-print(trainer.add_pokemon(second_pokemon))
-print(trainer.add_pokemon(second_pokemon))
-print(trainer.add_pokemon(second3_pokemon))
-print(trainer.add_pokemon(second4_pokemon))
-print(trainer.add_pokemon(second5_pokemon))
-print(trainer.release_pokemon("Pikachu"))
-print(trainer.release_pokemon("Pikachu"))
-print(trainer.trainer_data())
+
+# pokemon = Pokemon("Pikachu", 90)
+# print(pokemon.pokemon_details())
+# trainer = Trainer("Ash")
+# print(trainer.add_pokemon(pokemon))
+# second_pokemon = Pokemon("Charizard", 110)
+# second3_pokemon = Pokemon("Charizard3", 330)
+# second4_pokemon = Pokemon("Charizard4", 440)
+# second5_pokemon = Pokemon("Charizard5", 555)
+# print(trainer.add_pokemon(second_pokemon))
+# print(trainer.add_pokemon(second_pokemon))
+# print(trainer.add_pokemon(second3_pokemon))
+# print(trainer.add_pokemon(second4_pokemon))
+# print(trainer.add_pokemon(second5_pokemon))
+# print(trainer.release_pokemon("Pikachu"))
+# print(trainer.release_pokemon("Pikachu"))
+# print(trainer.trainer_data())
